gitlab/resource_gitlab_managed_license.go

```diff
@@ -0,0 +1,138 @@
+package gitlab
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/xanzy/go-gitlab"
+	"log"
+	"strconv"
+)
+
+func resourceGitlabManagedLicense() *schema.Resource {
+	return &schema.Resource{
```

Comment:
Please add a proper `Importer` in addition to the other CRUD functions.

Reply:
@timofurrer Are you ok with the `Importer` below? Please, resolve this conversation.

Reply:
Yes, I am. Since this was a solve comment I can't resolve it (no resolve button here 🤷 ). 

Consider it, as okay :)